src/components/NavBar.jsx

```diff
@@ -1,33 +1,56 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
 import PropTypes from "prop-types";
-import React from "react";
+import React, { useState } from "react";
 import FacebookLogin from "react-facebook-login";
 import { Link } from "react-router-dom";
-import "../styles/navbar.css";
+import "../styles/sass/navbar.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBars } from "@fortawesome/free-solid-svg-icons";
 
 const NavBar = ({ handleLogin, userID, handleLogout }) => {
+  const initialState = {
+    isActive: false,
+  };
+  const [isActive, setActive] = useState(initialState.isActive);
+
+  const handleToggleNav = () => {
+    if (isActive) {
+      setActive(false);
+    } else {
+      setActive(true);
+    }
+  };
+
   return (
     <div className="navbar">
+      <div
+        className="toggle"
+        onClick={handleToggleNav}
+        onKeyDown={handleToggleNav}
+      >
+        <FontAwesomeIcon icon={faBars} />
+      </div>
       <img
-        className="navbar__logo"
+        className="logo"
         src="https://mcrcodes.s3.eu-west-2.amazonaws.com/course/surreal-estate/logo.png"
         alt="siteLogo"
       />
-      <ul className="navbar-links">
-        <li className="navbar-links-item">
+      <ul className="links">
+        <li className="item">
           <Link className="item" to="/">
             View Properties
           </Link>
         </li>
         {userID ? (
-          <li className="navbar-links-item">
+          <li className="item">
```

Comment:
Looking at this ternary on line 44, would it not be better to use the AND operator? so
```
userID && (
  // your markup here
)
```

You wouldnt need the empty string return on line 51 then